cmake/modules/RootNewMacros.cmake

```diff
@@ -1079,6 +1079,22 @@ function(ROOT_ADD_TEST test)
 endfunction()
 
 #----------------------------------------------------------------------------
+# function ROOT_ADD_GTEST(<testsuite> <name> LIBRARIES)
+#
+function(ROOT_ADD_GTEST test_suite test_name)
```

Comment:
Is `test_name` `TQObjectTests.cxx` in the example below? If so, can you rename this to `source` then? And can we have multiple sources?